tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -129,6 +129,48 @@ def load_dut_basic_facts(session):
 
     return results
 
+def load_feature_status_facts(session):
+    """Run 'ansible -m get_feature' command to get some basic DUT feature status.
+
+    The facts will be a 1 level dictionary. The dict keys can be used as variables in condition statements evaluation.
+
+    Args:
+        session (obj): The pytest session object.
+
+    Returns:
+        dict or None: Return the dut feature status dict or None if something went wrong.
+    """
+    results = {}
+    logger.info('Getting dut feature status')
+    try:
+        testbed_name = session.config.option.testbed
+        testbed_file = session.config.option.testbed_file
+
+        testbed_module = imp.load_source('testbed', 'common/testbed.py')
+        tbinfo = testbed_module.TestbedInfo(testbed_file).testbed_topo.get(testbed_name, None)
+
+        results['topo_type'] = tbinfo['topo']['type']
+        results['topo_name'] = tbinfo['topo']['name']
+
```

Comment:
Is it better to add a new sub function to load testbed info, then `load_dut_basic_facts`
 can use it as well? It can save time to avoid loading it twice.

Reply:
I move the duplicate code into `load_basic_facts`, thus they are executed only once.